common/src/main/java/com/datatorrent/common/util/FSStorageAgent.java

```diff
@@ -71,12 +71,10 @@ public FSStorageAgent(String path, Configuration conf)
 
```

Comment:
@tushargosavi  this was causing lot of checkstyle violations and causing the build to fail so fixed it. 
sure will just revert it.